src/gui/optionsdlg.ui

```diff
@@ -2782,6 +2782,22 @@ Use ';' to split multiple entries. Can use wildcard '*'.</string>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
+                  <widget class="QLabel" name="lblWebUiAddress">
+                   <property name="text">
+                    <string>IP Address:</string>
+                   </property>
+                  </widget>
+                 </item>
+                 <item>
+                  <widget class="QLineEdit" name="textWebUiAddress">
+                   <property name="toolTip">
+                    <string>IP Address that the Web UI will bind to.
```

Comment:
I don't think "Address" should be capitalized.

Reply:
Also please move this QHBoxLayout (`horizontalLayout_2`) above "Server domain" layout and remove the spacer, as so:
![pic](https://user-images.githubusercontent.com/9395168/31557458-590aff64-b07b-11e7-8c84-b4a2d58a563a.png)